packages/web/src/scss/components/Item/_Item.scss

```diff
@@ -0,0 +1,85 @@
+@use 'sass:map';
+@use 'theme';
+@use '../../settings/cursors';
+@use '../../tools/typography';
+
+.Item {
+    display: grid;
+    grid-template-columns: auto 1fr auto;
+    padding: theme.$padding;
+    border-radius: theme.$border-radius;
+    background-color: theme.$background-default;
+    cursor: pointer;
+
+    @media (hover: hover) {
+        &:hover {
+            text-decoration: none;
+            background-color: theme.$background-hover;
+        }
+
+        &:active {
```

Comment:
Secondly, we decided not to limit the active state for hover-able devices only, at least I think so.

Anyway, the active state must be outside of the media query, otherwise we may get this active state on non-hover-able devices:

![image](https://github.com/lmc-eu/spirit-design-system/assets/5614085/f18ea26d-e598-4c28-9ce5-e142d6854a5b)